core/domain/exp_services.py

```diff
@@ -322,6 +416,14 @@ def get_exploration_ids_matching_query(query_string, cursor=None):
     there are at least that many, otherwise it returns all remaining results.
     (If this behaviour does not occur, an error will be logged.) The method
     also returns a search cursor.
+
+    Args:
+        query_string: A search query string.
```

Comment:
query_string: str. A search query string.
cursor: str or None. Optional cursor from which to start the search query. If no cursor is supplied, the first N results matching the query are returned.